net/nextdns/files/nextdns.config

```diff
@@ -0,0 +1,18 @@
+config nextdns main
+    option enabled '1'
```

Comment:
I suggest defaulting to disabled, especially since the user needs to add their configuration ID first.

Reply:
It works as a non-blocking DoH resolver with no config.

Reply:
I guess this is just personal preference, but I prefer to have services default to disabled so that nothing starts without my knowledge (and before I have a chance to review the configuration).

I don't think there are any general guidelines on this, so again this is just my opinion.

Reply:
I understand. My only concern is that it becomes two actions to set it up, you have to set a configuration ID and not forget (or miss) to check the "enabled" checkbox.

Reply:
Fair enough :slightly_smiling_face: